CONTRIBUTING.md

```diff
@@ -11,15 +11,17 @@ We welcome everyone to work on this cool project. We use a typical PR system:
 ### Visual Studio Code
 
 1. Clone your fork to your local machine
-2. Open the repository up in Visual Studio Code
-3. Connect to a remote IBM i where the build can happen
-4. Correctly set your current library to where your objects can be built
+2. Copy the the relevant c_cpp_properties.json.* file for your operating system to c_cpp_properties.json
+3. Run the downloadIbmiHeader utility on the ile folder to download the relevant headers to your local system: https://github.ibm.com/ibmi-toronto/downloadIBMiHeader
+4. Open the repository up in Visual Studio Code
+5. Connect to a remote IBM i where the build can happen
+6. Correctly set your current library to where your objects can be built
```

Comment:
What is the user's current library for? The objects are built into what we set as `BUILDLIB`.

Reply:
That line isn't even part of this pr, it was there before. I guess it may just be referring to building objects in general, not neccessarily specific to this project...